src/SessionSetup/PersistentStorage.cpp

```diff
@@ -40,4 +43,24 @@ void SaveProjectToPersistentStorage(std::optional<Project> project) {
   }
 }
 
+[[nodiscard]] InstanceListScope LoadInstancesScopeFromPersistentStorage() {
+  QSettings settings;
+  if (settings.contains(kInstancesScopeKey)) {
+    return InstanceListScope::kAllReservedInstances;
+  }
+  return InstanceListScope::kOnlyOwnInstances;
+}
+
+void SaveInstancesScopeToPersistentStorage(InstanceListScope scope) {
+  QSettings settings;
+  switch (scope) {
+    case InstanceListScope::kOnlyOwnInstances:
+      settings.remove(kInstancesScopeKey);
```

Comment:
I recommend not to use "key does not exist" as a valid setting here. Because that means you can't decide whether the user deliberately chose `kOnlyOwnInstances` the last time when they started Orbit or if it just the default.

Right now that makes no difference but maybe we wanna change that behaviour in the future and then it's valuable to know if the default should apply (the user never started Orbit) or if we should migrate the user's last choice to the equivalent of the new behaviour.

TLDR: I recommend to use two strings for the enum values. That also allows to add more scopes in the future if needed.

Reply:
Ahh I just saw that this is not new code and the setting already existed this way? If that's the case and we can't break it feel free to ignore me on this.

Reply:
> Because that means you can't decide whether the user deliberately chose kOnlyOwnInstances the last time when they started Orbit or if it just the default.

But this is exactly what I want. At this point I really dont care if it was chosen deliberately, or if its the default. 

I looked a while into making this more sustainable in the sense of saving enum values to QSettings. I couldn't find an elegant and simple solution. Q_ENUMS would probably the best way, but its again more work for enums that are not part of a QObject. 

I decided to go this simple and not so elegant way and am aware of the tech debt, aka that adding more scopes in the future is slightly more work.